src/olympia/users/admin.py

```diff
@@ -169,7 +173,24 @@ def reset_api_key_view(self, request, object_id, extra_context=None):
         if not acl.action_allowed(request, amo.permissions.USERS_EDIT):
             return HttpResponseForbidden()
 
-        self.reset_api_key_action(request, [obj])
+        self.reset_api_key_action(
+            request, UserProfile.objects.filter(pk=obj.pk))
+
+        return HttpResponseRedirect('../')
```

Comment:
ha?

Reply:
I see this used already but returning the full URL seems a bit more future-proof to me.

Reply:
https://github.com/mozilla/addons-server/pull/13763/commits/20814c702404863bbb56f2e2b73d8d609b1791f9